src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -40,8 +40,10 @@ public class AddCommand extends Command {
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_DATE_OF_BIRTH + "9 Mar 1999"
             + PREFIX_PRIORITY + "HIGH "
             + PREFIX_REMARK + "Promises to pay back next week. "
+            + PREFIX_INCOME + "2000"
```

Comment:
There should be a whitespace after `dob/9 Mar 1999` and `income/2000`.

Reply:
Noted. Thanks for catching the mistake